### DISCUSSION TITLE
Can I tell from the server side if there is a remote connection to a display?

### DISCUSSION THREAD

**Discussion by jzyrobert** [Xpra-org/xpra#3817] [Q&A] [answered]
Using Xpra 4.4.4, I have:
- Ubuntu server with an active Xpra display
- Windows client connected to that display over SSH

On the Ubuntu side, can I use one of the commands to tell me if the windows client is connected or not?
`xpra list/list-sessions` seems to show the same output whether the client is connected to not.
I had a look at `xpra info` but didn't finjd anything obvious.

**Comment by totaam** [comment, marked as answer]
```shell
$ xpra info | grep clients=
```
You can also see the client(s) connected using `xpra top`.

**Reply by jzyrobert** [reply]
Thanks!
